### PR TITLE
New Component: mate-media

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

**Comment by pyhalov** [comment]
Note to committer: requires libmatemixer and mate-desktop, check their presence on the build server

**Comment by Toasterson** [comment]
Build server has caught up with prs libamtemixer and mate-desktop are now present ob buildserver

File: components/desktop/mate/mate-media/Makefile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/1.14/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE= GPLv2, LGPLv2, FDLv1.1
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION = System/Libraries
+COMPONENT_FMRI = library/desktop/mate/$(COMPONENT_NAME)
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
```
Comment (pyhalov) [inline components/desktop/mate/mate-media/Makefile]: Please, if component uses python, add PYTHON_VERSION=2.7

File: components/desktop/mate/mate-media/Makefile [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+COMPONENT_PREP_ACTION=	cd $(@D)  && NOCONFIGURE=1 ./autogen.sh
+
+CONFIGURE_OPTIONS+= --sysconfdir=/etc
+CONFIGURE_OPTIONS+= --libexecdir=/usr/lib/mate
+CONFIGURE_OPTIONS+= --disable-static
+CONFIGURE_OPTIONS+= --disable-scrollkeeper
+CONFIGURE_OPTIONS+= --localstatedir=/var/lib
+CONFIGURE_OPTIONS+= --with-gtk=2.0
+
+CONFIGURE_ENV+= PYTHON="$(PYTHON)"
```
Comment (pyhalov) [inline components/desktop/mate/mate-media/Makefile]: This component's configure script doesn't check for PYTHON, you can remove this line

**Pull request merged**